### ISSUE TITLE
Restore the ability for an active region to determine the quoted text in a reply

### ISSUE THREAD

**Issue by phil-s** [alphapapa/ement.el#275]
In https://github.com/alphapapa/ement.el/pull/273 I've removed the text from the readme's description of the 'reply' command which said "when region is active, only quote marked text", as this is no longer correct.

It was a very nice feature, though, so it would be nice to restore it!

https://github.com/alphapapa/ement.el/pull/150 may present another conflict for doing so.

**Comment by alphapapa** [comment]
Yeah, it seems that that functionality was lost accidentally.  I would like to restore it at some point.  It will have to be considered how to do so while also following the Matrix spec reasonably well, and accounting for rich reply support.  Thanks.
